test/api/junction_detection_methods_test.cpp

```diff
@@ -253,10 +253,10 @@ TEST(junction_detection, detect_junctions_in_long_reads_sam_file)
     Breakend new_breakend_8 {chromosome_1, pos_read_4, strand::forward, sequence_type::reference};
     Breakend new_breakend_9 {chromosome_1, pos_read_4, strand::forward, sequence_type::reference};
 
-    const std::string & read_name_1 = "m2257/8161/CCS";
-    const std::string & read_name_2 = "m41327/11677/CCS";
-    const std::string & read_name_3 = "m21263/13017/CCS";
-    const std::string & read_name_4 = "m38637/7161/CCS";
+    std::string const & read_name_1 = "m2257/8161/CCS";
+    std::string const & read_name_2 = "m41327/11677/CCS";
+    std::string const & read_name_3 = "m21263/13017/CCS";
+    std::string const & read_name_4 = "m38637/7161/CCS";
 
```

Comment:
These shouldn't be references. (You could make them std::string_view)
```suggestion
    std::string const read_name_1 = "m2257/8161/CCS";
    std::string const read_name_2 = "m41327/11677/CCS";
    std::string const read_name_3 = "m21263/13017/CCS";
    std::string const read_name_4 = "m38637/7161/CCS";

```